python_modules/dagster/dagster/_grpc/impl.py

```diff
@@ -240,19 +242,13 @@ def get_external_pipeline_subset_result(
 
 
 def get_external_schedule_execution(
-    recon_repo,
-    instance_ref,
-    schedule_name,
-    scheduled_execution_timestamp,
-    scheduled_execution_timezone,
+    repo_def: RepositoryDefinition,
+    instance_ref: Optional[InstanceRef],
```

Comment:
this seems clearer anyway even if there wasn't a perf benefit - unclear why we were doing this recon repo dance before